assets/javascript/admin-dashboard.js

```diff
@@ -0,0 +1,120 @@
+'use strict'
+import Chart from 'chart.js/auto'
```

Comment:
The other day I was thinking about trying out plotly / dash, [but for django](https://django-plotly-dash.readthedocs.io/en/latest/). I have worked with it before outside of django and it is pretty easy to build beautiful and powerful dashboards with graphs etc. Maybe also something to consider if we ever need to go bigger

Reply:
And the nice thing about it is that the code is mostly python

Reply:
Looks interesting